locales/en/messages.json

```
@@ -5752,7 +5752,7 @@
         "description": "One of the elements of the OSD"
     },
     "osdDescElementEscTemperature": {
-        "message": "Temperature reported by ESC telemetry"
+        "message": "Temperature reported by ESC telemetry (from ESC_SENSOR or DShot telemetry with EDT support enabled. Refer to your ESC firmware for EDT support)"
```

Maybe the user does not know what is ESC_SENSOR. I think is better to reduce the technical develop words. The user knows about the "one extra wire telemetry" and about the "bidirectional telemetry". My English is not good but something like:
```suggestion
        "message": "Temperature reported by ESC telemetry (from one extra wire telemetry or DShot bidirectional telemetry with EDT support enabled. Refer to your ESC firmware for EDT support)"
```